MoonbamiOfficial/sassy-css
# chore: add github actions deployment

# Type of Change

- [x] Chore (change for other operations)

###

# Description

Add a github actions for vercel deployment

###

# How Has This Been Tested?

1. Create a test branch, then commit and push changes to this branch. 
2. Check the github actions of this repo.

**Test Configuration**:

- Browser: Google Chrome 
- Browser version: 127.0.6533.89 (Official Build) (64-bit)
- Text editor: Visual Studio Code
